packages/styles/src/variant-management.scss

```diff
@@ -0,0 +1,58 @@
+@import "./mixins";
+
+$block: #{$fd-namespace}-variant-management;
+
+$title-margin: 0.125rem;
+
+.#{$block} {
+  @include fd-reset();
+
+  &__header {
+    @include fd-reset();
+
+    @include fd-flex-vertical-center() {
+      cursor: pointer;
```

Comment:
since you are using flex you can add `gap: 0.125rem` and then get rid of margin-right of the child elements